File: plusmall-operator-interface/src/main/java/com/plusmall/operator/TypeTempService.java
package com.plusmall.operator;

import com.plusmall.commons.ActionResult;
import com.plusmall.commons.PageResult;
import com.plusmall.model.TbTypeTemplate;

import java.util.List;
import java.util.Map;

/**
 * @Description:
 */
public interface TypeTempService {
	public PageResult search(int pageNum, int pageSize, TbTypeTemplate typeTemplate);
	public void delete(Long[] ids);
	public void add(TbTypeTemplate typeTemplate);
	public TbTypeTemplate findOne(Long id);
	public void update(TbTypeTemplate typeTemplate);
	public List<TbTypeTemplate> findAll();
	public List<Map> findSpecList(Long id);
}

File: plusmall-operator-web/src/main/webapp/js/service/typeTempService.js
app.service('typeTempService',function ($http) {

    //查找
    this.search=function (pageNum,pageSize,searchEntity) {
        return $http.post('../typeTemplate/search.do?pageNum='+pageNum+'&pageSize='+pageSize,searchEntity);
    }

    //删除
    this.delete=function (ids) {
        return $http.get('../typeTemplate/delete.do?ids='+ids);
    }

    //新增
    this.add=function (typeTemp) {
        return $http.post('../typeTemplate/add.do',typeTemp);
    }

    //根据Id查找
    this.findOne=function (id) {
        return $http.get('../typeTemplate/findOne.do?id='+id);
    }

    //更新
    this.update=function (typeTemp) {
        return $http.post('../typeTemplate/update.do',typeTemp);
    }
});